notebooks/data_mining.ipynb

```diff
@@ -153,14 +153,14 @@
      "name": "stderr",
```

Comment:
<p>Is this warning misleading?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>